### PR TITLE
add microformats2 markup to examples folder

### PR DESCRIPTION
**Pull request by aaronpk**
This PR adds a new example showing the same article with Microformats2 markup. You can see the [parsed result in JSON format](http://glennjones.net/tools/microformats/?url=https%3A%2F%2Fraw.githubusercontent.com%2Faaronpk%2Famphtml%2Fmaster%2Fexamples%2Farticle-metadata%2Fmicroformats.amp.html).

For more details and background on Microformats2, see the following links:
* http://microformats.org/wiki/h-entry
* http://microformats.org/2014/03/05/getting-started-with-microformats2

### CODE REVIEW

File: examples/article-metadata/microformats.amp.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        Also, please be aware that some platforms that use AMP HTML have
+        further restrictions with regards to some schema components.
+
+         For example:
+
+           * All marked-up URL's should be absolute.
+           * The "logo" dimensions must not exceed 600x60.
+    -->
+    <script src="https://cdn.ampproject.org/v0.js" async></script>
```
Comment (jmadler) [inline examples/article-metadata/microformats.amp.html]: This tag needs to be at the bottom of the head.  #458 is tracking that for other samples.

Comment (aaronpk) [inline examples/article-metadata/microformats.amp.html]: I'm confused about where this is supposed to go. The example here shows the CDN tag in the middle of the head and the noscript tag at the bottom of the head: https://github.com/ampproject/amphtml/blob/master/docs/create_page.md#write-the-basic-amp-html-page

Comment (jmadler) [inline examples/article-metadata/microformats.amp.html]: The <script> tag including the AMP lib's JS file is supposed to be the last element of the head.  It's not consistently documented currently, unfortunately.

I've created #503 to address that.

Comment (aaronpk) [inline examples/article-metadata/microformats.amp.html]: okay thanks, updated this example

**Comment by jmadler** [comment]
Awesome!  @cramforce Are you ok with including microformats2 in the spec as an acceptable structured data markup format?

**Comment by cramforce** [comment]
Could you squash your commit?

Not sure this really needs an example file yet.

CC @danbri

**Comment by danbri** [comment]
Is anyone consuming microformats-2 yet, at scale?

**Comment by aaronpk** [comment]
@cramforce I just updated the PR to include just one commit. 

@danbri The largest producer of microformats2 is probably https://www.brid.gy right now. Bridgy is used to send comments from Facebook, Twitter and other silos to websites that consume Microformats2, which it does by creating an h-entry page for each Tweet, Facebook comment, Flickr comment, etc.

There are user stats at the bottom of the Bridgy home page. Across 2,394 users there have been over 2 million h-entry pages consumed. These were of course consumed by many different implementations rather than a single service, so that fits at least one definition of "scale".

**Comment by danbri** [comment]
Thanks @aaronpk. I was asking more about the consumption side; I guess brid.gy is a little of both, since it consumes non-microformats2 from Facebook/Twitter/etc., and publishes it to other sites. Are there substantial sites that consumer microformats2 data natively?

**Comment by kevinmarks** [comment]
Are there any sites that consume json-ld or microdata natively?

**Comment by aaronpk** [comment]
Probably the largest single consumer of microformats2 right now is http://withknown.com. Their comments are powered by microformats2 and webmention. 

There is also a wordpress plugin that consumes microformats2 for comments, although I don't know how many sites it is deployed on obviously.

**Comment by erwinmombay** [comment]
@aaronpk mind rebasing and pushing again. thanks

**Comment by aaronpk** [comment]
Sure thing!

**Comment by erwinmombay** [comment]
@aaronpk appreciate it :) CI is green now too. apologies for the flaky tests :(

**Comment by cramforce** [comment]
TBH I'd much rather just delete all these examples. AMP doesn't really care about how docs are marked up. In any case we'd only would want to demonstrate the most popular ones.

I will also remove almost the entirety of the schema.org requirements from the spec, but I will keeo the requirement to state the type of the page, because that will be very useful in the future to get rid of AMP :)

**Comment by amplesample** [comment]
I think we should only support microdata and JSON-LD. Microformats are an older format, and it doesn't make sense to use these for AMP.

**Comment by aaronpk** [comment]
I am not sure the comment above is helpful. This account was opened less than a month ago, has no followers and no following/starred repos, no repos of their own, and the only recent public activity is this comment. 

Additionally, the statement is based on out of date information. Microformats2 is the update to Microformats1, and has been developed taking into account the lessons learned and features used in practice from both microdata and RDFa, which is documented here: http://microformats.org/wiki/microformats2

**Comment by cramforce** [comment]
@aaronpk Please refrain from personal attacks on this Github project.

We decided to remove metadata requirements from AMP. See #593

We will rework examples to a single "common practice" example that has metadata markup that would be commonly used by a publisher in late 2015.

**Comment by kevinmarks** [comment]
You know what would be great to determine common practice? Re-running something like Hixie's Web Authoring Stats survey from a  decade ago:

https://web.archive.org/web/20080729053114/http://code.google.com/webstats/index.html

**Pull request closed without merging**